tests/test_uniform.py

```diff
@@ -27,6 +27,26 @@ def test_uopengraph(self):
         data = extruct.extract(body, syntaxes=['opengraph'], uniform=True)
         self.assertEqual(data['opengraph'], expected)
 
+    def test_uopengraph_with_og_array(self):
+        expected = [{"@context": {
+                        "og": "http://ogp.me/ns#",
+                        "fb": "http://www.facebook.com/2008/fbml",
+                        "concerts": "http://ogp.me/ns/fb/songkick-concerts#"
+                    },
+                    "fb:app_id": "308540029359",
+                    "og:site_name": "Songkick",
+                    "@type": "songkick-concerts:artist",
+                    "og:title": "Elysian Fields",
+                    "og:description": "Buy tickets for an upcoming Elysian Fields concert near you. List of all Elysian Fields tickets and tour dates for 2017.",
+                    "og:url": "http://www.songkick.com/artists/236156-elysian-fields",
+                    "og:image": [ "http://images.sk-static.com/3.jpg",
+                                  "http://images.sk-static.com/2.jpg",
+                                  "http://images.sk-static.com/1.jpg"],
+                }]
+        body = get_testdata('misc', 'ogarray_test.html')
+        data = extruct.extract(body, syntaxes=['opengraph'], uniform=True, with_og_array=True)
```

Comment:
Thanks for adding a test! Could you please also add a check how we handle the same input data with ``with_og_array``?

Reply:
> Could you please also add a check how we handle the same input data with with_og_array?

Do you mean a check to see how this data is handled when `with_og_array` is False ?

Reply:
> Do you mean a check to see how this data is handled when with_og_array is False ?

Yes, exactly - the reason why I think it's valuable is that we could check that we take the first out of multiple values for an attributes, and it would help in case of future development here. Thank you!

Reply:
Actually there already exists a [test](https://github.com/scrapinghub/extruct/blob/master/tests/test_uniform.py#L12) for multiple values for an attribute. It's just that here I needed more than 2 values otherwise it was failing codecov. I can add this test if you want me to.

Reply:
I see, thank - probably you mean this https://github.com/scrapinghub/extruct/blob/master/tests/test_uniform.py#L30 ? I think it's enough, thank you!

Reply:
Actually I've added the test that you mentioned with setting`with_og_array=True` as well. This would make that additional file that I added useless so I've removed that.